src/Components/Classes/Fprm/newclass.module.css

```diff
@@ -0,0 +1,104 @@
+.addClass {
+  width: 70vh;
+  height: 10vh;
+  padding-top: 30px;
+}
+.updateClass {
+  width: 70vw;
+  height: 46vh;
+}
+.formControl,
+.formControlUpdate {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  align-items: center;
+  background-color: #0f232e;
+}
+.formControlUpdate {
+  background-color: #0f232e;
+}
+.formControlUpdate label {
+  width: 6vw;
+  padding: 15px 10px;
+  font-size: 1.3rem;
+  text-align: inherit;
+}
+.formControlUpdate select {
+  width: 15vw;
+  height: 2vh;
+  margin-left: 20px;
+}
+.formControlUpdate input {
+  margin-left: 20px;
+}
+label {
```

Comment:
Try to avoid using universal selectors, as they will override styles for all elements

Reply:
fixed